Dockerfile

```diff
@@ -9,5 +9,4 @@ COPY ./requirements.txt /app/requirements.txt
 WORKDIR /app
 RUN apt-get update && apt-get install --no-install-recommends -y python-numpy==1.16.5 build-essential==12.9 && rm -rf /var/lib/apt/lists/*
 ENV CFLAGS="-I /usr/local/lib/python3.8/site-packages/numpy/core/include $CFLAGS"
-RUN pip install --no-cache-dir -r requirements.txt
-RUN cython nicenlp/utils/greynir/tree_dist.pyx
+RUN pip install --no-cache-dir -r requirements.txt && cython nicenlp/utils/greynir/tree_dist.pyx
```

Comment:
Vorum við ekki búnir að fjarlægja `requirments.txt`?

Reply:
Yes, this dockerfile needs rewriting.